WordPress/Classes/ViewRelated/Gutenberg/CollapsableHeader/CollapsableHeaderViewController.xib

```diff
@@ -189,9 +187,7 @@
                                 <color key="backgroundColor" systemColor="systemBackgroundColor"/>
                             </view>
                             <color key="tintColor" systemColor="systemBackgroundColor"/>
-                            <vibrancyEffect style="fill">
-                                <blurEffect style="systemChromeMaterial"/>
-                            </vibrancyEffect>
```

Comment:
This change looks good to me 👍 We account for the styling [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/e0c636b15b5987976f7cd4a1ce076fdbaaa66f49/WordPress/Classes/ViewRelated/Gutenberg/Collapsable%20Header/CollapsableHeaderViewController.swift#L41). Thanks for taking care of this!